_redirects

```diff
@@ -79,3 +79,10 @@
 
 # 2024 Welcome
 /welcome https://www.youtube.com/watch?v=gJcnfB2ep7E
+
+# 2024 BGN
+/bgn https://forms.gle/EoxFaFMbHGCANTCF9
+/bringing-games https://forms.gle/EoxFaFMbHGCANTCF9
+/board-games https://docs.google.com/spreadsheets/d/1r4Zgx7czXcZuh93P-5vxM-3IK8vC2xeZ04g0zjiNb_g/edit?gid=99133481#gid=99133481
+/bgn-games https://docs.google.com/spreadsheets/d/1r4Zgx7czXcZuh93P-5vxM-3IK8vC2xeZ04g0zjiNb_g/edit?gid=99133481#gid=99133481
```

Comment:
Aliases of the same destination should redirect to each other rather than duplicating the link